pkg/virt-launcher/virtwrap/network/dhcp/dhcp.go

```diff
@@ -91,6 +93,12 @@ func SingleClientDHCPServer(
 	}
 	dhcpOptions[dhcp.OptionHostName] = []byte(hostname)
 
+	for _, customDHCPOption := range customDHCPOptions {
+		log.Log.Infof("Setting dhcp option %s to %s", customDHCPOption.OptionCode, customDHCPOption.OptionValue)
+		optionCode := dhcp.OptionCode(customDHCPOption.OptionCode)
+		dhcpOptions[optionCode] = []byte(customDHCPOption.OptionValue)
```

Comment:
I guess that works for a lot of options. At least for the search domains array it would not work.

Reply:
I plan on adding a conditional that doesn't allow any options kubevirt set to be overriden.

Reply:
Search domain is an option kubevirt sets so it would be excluded from this.